ClinGen/clincoded
# Connects to #1406. Additional genotype methods in GCI.

**Steps to test:**
1. In GDM, start a new Group/Family/Individual curation.
2. on the curation form page, scroll down to the *** - Methods** panel.
3. Under the **Genotyping Method** header in the panel, click on the **Method 1** dropdown and expect to see the following list:
![screen shot 2018-07-26 at 6 42 42 pm](https://user-images.githubusercontent.com/15131169/43297736-d6abda8e-9107-11e8-9a25-945ce6e30b19.png)
4. Select any one of the options from the list, and then click on the **Method 2** dropdown. Expect to see the same list shown above as well.
5. Select any one of the options and click the **Save** button at the bottom. Expect to be able to submit the form successfully.
